tests/experiment/test_model.py

```diff
@@ -419,3 +446,18 @@ def test_dict_round_trip(interface):  # , additional_layer):
             model.interface().fit_func([0.3], model.unique_name),
             model_from_dict.interface().fit_func([0.3], model_from_dict.unique_name),
         )
+
+
+def test_dict_skip_unique_name():
```

Comment:
The `skip` argument is being tested elsewhere. Is there a reason to explicitly test this here?

Reply:
Moved to `test_topmost_nesting.py`.  These are test to ensure that the nested structure is functional by using a method from the topmost class